package.json

```diff
@@ -1,56 +1,50 @@
 {
   "name": "eslint-config-molindo",
-  "version": "7.0.0",
+  "version": "8.0.0",
   "description": "Molindo ESLint config that implements our styleguide and helps to catch errors.",
   "repository": "https://github.com/molindo/eslint-config-molindo",
   "author": "Jan Amann <jam@molindo.at>",
+  "type": "module",
   "license": "MIT",
+  "main": "src/index.js",
   "keywords": [
     "eslint",
     "eslintconfig"
   ],
   "scripts": {
-    "lint": "eslint .",
-    "test": "jest",
-    "prepublish": "npm run lint && npm run test"
+    "lint": "eslint src",
+    "prepublish": "npm run lint"
   },
   "files": [
-    "setupPlugins.js",
-    "javascript.js",
-    "typescript.js",
-    "config.js",
-    "css-modules.js",
-    "tailwind.js",
-    "react.js",
-    "jest.js",
-    "cypress.js",
+    "src",
+    ".prettierrc.json",
     "tsconfig.json"
   ],
   "devDependencies": {
-    "eslint": "^8.45.0",
-    "jest": "^29.6.2",
-    "typescript": "5.1.6"
+    "eslint": "^9.0.0",
+    "prettier": "^3.3.0",
+    "tailwindcss": "^3.4.13",
+    "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "eslint": "^8.0.0"
+    "eslint": "^9.0.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.2.0",
-    "@typescript-eslint/parser": "^6.2.0",
+    "@eslint/js": "^9.11.1",
+    "@typescript-eslint/utils": "^8.0.0",
+    "@vitest/eslint-plugin": "^1.1.7",
     "confusing-browser-globals": "^1.0.11",
+    "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-css-modules": "^2.11.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.2.3",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-react": "^7.33.0",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-sort-destructure-keys": "^1.5.0",
-    "eslint-plugin-tailwindcss": "3.13.0",
-    "eslint-plugin-unicorn": "^48.0.1",
-    "prettier": "^3.0.0"
-  },
-  "engines": {
-    "node": ">=10"
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jest": "^28.2.3",
+    "eslint-plugin-jsx-a11y": "^6.10.0",
+    "eslint-plugin-react": "^7.37.0",
+    "eslint-plugin-react-hooks": "5.1.0-beta-26f2496093-20240514",
```

Comment:
To get rid of a warning during install.

Reply:
- [ ] TODO: Use [release version](https://github.com/facebook/react/releases/tag/eslint-plugin-react-hooks%405.0.0)